spec/spec_helper.rb

```diff
@@ -1,16 +1,15 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV['RAILS_ENV'] ||= 'test'
-require 'factory_bot_rails'
```

Comment:
See explanation for changes in this file here: 25ac30ec2d34c7f56d69c84aad285ea61331bb5c